rtf_vis_tool/src/Components/gtsfm_edge_list.js

```diff
@@ -0,0 +1,58 @@
+//List of all Edges in GTSFM Visualization Graph
```

Comment:
rename to gtsfm_graph

// Saves the bipattite graph (U,V,E), where U are the (gray) products, and (V) are the (blue) processes.

Reply:
I ended up having to store the graph in 3 separate JS files, because in any file, I can only export one variable. So, instead, I have created a directory called `gtsfm_graph`, and inside that directory, I have:

- `gray_nodes.js`: stores all the gray nodes
- `blue_nodes.js`: stores all the blue nodes
- `edge_list.js`: stores all the directed edges